server/TypeDBServer.java

```diff
@@ -233,18 +237,18 @@ public static void main(String[] args) {
             printASCIILogo();
 
             CoreConfigParser configParser = new CoreConfigParser();
-            ArgsParser<ServerSubcommand> argsParser = new ArgsParser<ServerSubcommand>()
-                    .subcommand(new ServerSubcommandParser.Server(configParser))
-                    .subcommand(new ServerSubcommandParser.Import())
-                    .subcommand(new ServerSubcommandParser.Export());
-            Optional<ServerSubcommand> subcmd = argsParser.parse(args);
+            ArgsParser<CoreSubcommand> argsParser = new ArgsParser<CoreSubcommand>()
```

Comment:
We follow the existing naming convention where anything called `Server` is shared by Core and Cluster, otherwise we use the `Core` and `Cluster` prefixes. This change is caused by Cluster needing to extend command-line commands and not use shared bootup commands anymore